setup.py

```diff
@@ -51,7 +51,7 @@ def _read_contents(file_to_read):
     packages=packages,
     install_requires=[
         'aenum;python_version<"3.6"',
-        "events",
+        "events==0.3",
```

Comment:
Note: I think [this change](https://github.com/pyeve/events/commit/e1895e27a6f87417fb87633d444d5a2562994b16) broke a few tests that were asserting on instances of `HttpTagSubscription` so locking down the version to `0.3` for now.

Reply:
Hmm. That's a weird behavior. I think a better fix may be to define our own `__bool__` method that returns `True`, and/or update the tests to "assert uut is not None" instead of just "assert uut".